tools/lib.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+# Copyright (c) 2023 Petr Vorel <pvorel@suse.cz>
+
+ask()
+{
+	local msg="$1"
+	local answer
+
+	printf "\n$msg. Proceed? [N/y]: "
+	read answer
+	case "$answer" in
+		[Yy]*) : ;;
+		*) exit 2
+	esac
+}
+
+quit()
+{
+	printf "\n$@\n" >&2
+	exit 1
+}
+
+rod()
```

Comment:
i have no idea what this is short for.  func (all of them really) could use a short comment header.

Reply:
+1. Inspired by LTP, I wonder myself.

Reply:
The best I could come up with when reading this was that it's short for "run-or-die".  But yeah, some more descriptive names and/or descriptive comments would be nice.

Reply:
Yep, it is "run or die". I added descriptive comments, although it's a bit overkill for 1 or 2 lines script :).

Reply:
it's easy to write bad shell.  but bad shell is also bad ;).